lib/passwordless/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Passwordless
-  VERSION = '0.7.0' # :nodoc:
+  VERSION = "0.7.0" # :nodoc:
```

Comment:
Please run `rubocop` and make sure to align.

Reply:
Not sure how this snuck in :smile:

I notice you have `standardrb` shall I use that?

Reply:
Yes please. Not sure if hound knows that but we’ll find out. Standard is right and hound should probably be disabled.